components/collection/CollectionInfo.vue

```diff
@@ -19,15 +19,15 @@
         @click.native="toggleSeeAllDescription" />
     </div>
     <div>
-      <div class="columns is-mobile">
-        <div class="column">
+      <div class="columns pt-3">
```

Comment:
I wouldn't add padding on columns/column, wrap it in a new div instead

Reply:
In this component, I need to overwrite some padding on `column`, so I guess I wrap it in a new div. wdyt

Reply:
Please don't add spacing in columns/column class. If you want to add spacing then create a inner div that wrap content.
If you don't want spacing you can use [gapless](https://bulma.io/documentation/columns/gap/#gapless) helper or change columns to flex.